### PR TITLE
TCPServer: explain how to get an ephermal port

### PR DESCRIPTION
**Pull request by rdp**
Explain how to be able to receive an ephermal port ("magic number" 0)

### CODE REVIEW

File: src/socket/tcp_server.cr [hunk 1]
```diff
@@ -18,7 +18,9 @@ require "./tcp_socket"
 # ```
 #
 # Options:
-# - *backlog* to specify how many pending connections are allowed;
+# - *host* local interface to bind on, or `::` to bind on all local interfaces.
+# - *port* specific port to bind on, or 0 to receive an "ephemeral" (free, assigned by kernel) port.
```
Comment (Sija) [inline src/socket/tcp_server.cr:22]:
```suggestion
# - *port* specific port to bind on, or `0` to receive an "ephemeral" (free, assigned by kernel) port.
```

**Review by straight-shoota — APPROVED** [review]

**Review by oprypin — APPROVED** [review]

File: src/socket/tcp_server.cr [hunk 2]
```diff
@@ -18,7 +18,9 @@ require "./tcp_socket"
 # ```
 #
 # Options:
-# - *backlog* to specify how many pending connections are allowed;
+# - *host* local interface to bind on, or `::` to bind on all local interfaces.
+# - *port* specific port to bind on, or `0` to receive an "ephemeral" (free, assigned by kernel) port.
```
Comment (yxhuvud) [inline src/socket/tcp_server.cr:22]: Does it work the same way on BSD/Darwin and on Windows?

Comment (straight-shoota) [reply src/socket/tcp_server.cr:22]: Yes. All supported operating systems' implementations are based on the BSD socket API.

**Review by sdogruyol — APPROVED** [review]
Thank you @rdp 🙏

**Pull request merged**